src/molecules/shirt.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PricedColourButton from '../atoms/pricedColourButton';
+
+function Shirt(props) {
+
+  const [color, setColor] = useState();
```

Comment:
change value of this to `useStste(colours[0])`
and move colours array before this line